clients/ui/frontend/src/__tests__/cypress/cypress/tests/mocked/modelVersionArchive.cy.ts

```diff
@@ -185,119 +188,124 @@ describe('Model version archive list', () => {
-//     restoreVersionModal.findRestoreButton().click();
-
-//     cy.wait('@versionRestored').then((interception) => {
-//       expect(interception.request.body).to.eql({
-//         state: 'LIVE',
-//       });
-//     });
-//   });
-// });
-
-// describe('Archiving version', () => {
-//   it('Archive version from versions table', () => {
-//     cy.interceptApi(
-//       'PATCH /api/service/modelregistry/:serviceName/api/model_registry/:apiVersion/model_versions/:modelVersionId',
-//       {
-//         path: {
-//           serviceName: 'modelregistry-sample',
-//           apiVersion: MODEL_REGISTRY_API_VERSION,
-//           modelVersionId: 3,
-//         },
-//       },
-//       mockModelVersion({}),
-//     ).as('versionArchived');
-
-//     initIntercepts({});
-//     modelVersionArchive.visitModelVersionList();
-
-//     const modelVersionRow = modelRegistry.getModelVersionRow('model version 3');
-//     modelVersionRow.findKebabAction('Archive model version').click();
-//     archiveVersionModal.findArchiveButton().should('be.disabled');
-//     archiveVersionModal.findModalTextInput().fill('model version 3');
-//     archiveVersionModal.findArchiveButton().should('be.enabled').click();
-//     cy.wait('@versionArchived').then((interception) => {
-//       expect(interception.request.body).to.eql({
-//         state: 'ARCHIVED',
-//       });
-//     });
-//   });
-
-//   it('Archive version from versions details', () => {
-//     cy.interceptApi(
-//       'PATCH /api/service/modelregistry/:serviceName/api/model_registry/:apiVersion/model_versions/:modelVersionId',
-//       {
-//         path: {
-//           serviceName: 'modelregistry-sample',
-//           apiVersion: MODEL_REGISTRY_API_VERSION,
-//           modelVersionId: 3,
-//         },
-//       },
-//       mockModelVersion({}),
-//     ).as('versionArchived');
-
-//     initIntercepts({});
-//     modelVersionArchive.visitModelVersionDetails();
-//     modelVersionArchive
-//       .findModelVersionsDetailsHeaderAction()
-//       .findDropdownItem('Archive version')
-//       .click();
-
-//     archiveVersionModal.findArchiveButton().should('be.disabled');
-//     archiveVersionModal.findModalTextInput().fill('model version 3');
-//     archiveVersionModal.findArchiveButton().should('be.enabled').click();
-//     cy.wait('@versionArchived').then((interception) => {
-//       expect(interception.request.body).to.eql({
-//         state: 'ARCHIVED',
-//       });
-//     });
-//   });
-// });
+describe('Restoring archive version', () => {
+  it('Restore from archive table', () => {
+    cy.interceptApi(
+      'PATCH /api/:apiVersion/model_registry/:modelRegistryName/model_versions/:modelVersionId',
+      {
+        path: {
+          modelRegistryName: 'modelregistry-sample',
+          apiVersion: MODEL_REGISTRY_API_VERSION,
+          modelVersionId: 2,
+        },
+      },
+      mockBFFResponse(mockModelVersion({})),
```

Comment:
Notice how we have to put a bunch of `mockBFFResponse` everywhere. We should figure out a way for all of our `mockObject` calls to inherently know about the BFF response so we don't have to be so explicit.

Reply:
This is a great point, my first guess is to just modifify the mock objects and add a wrapper there, can you open a ticket in enhancemets to cover this?